ui/test/layouts/search/test/nuxeo-test-search-form.html

```diff
@@ -0,0 +1,81 @@
+<!--
+`nuxeo-test-search-form`
+@group Nuxeo UI
+@element nuxeo-test-search-form
+-->
+<dom-module id="nuxeo-test-search-form">
+  <template>
+    <style include="nuxeo-styles"></style>
+
+    <nuxeo-input
+      role="widget"
+      id="searchInput"
+      type="search"
+      label="[[i18n('defaultSearch.fullText')]]"
+      name="Fulltext"
```

Comment:
super nit: for consistency, shouldn't it be lowercase?
```suggestion
      name="fulltext"
```

Reply:
I'm noticing that this attribute isn't actually used anywhere, so I'm just going to remove it.